resources/openshift/templates/datavirt-app-secret.yaml

```diff
@@ -0,0 +1,150 @@
+kind: List
+apiVersion: v1
+metadata:
+  annotations:
+    description: Examples that can be installed into your project to allow you to test the JBoss Data Virtualization templates. You should replace the contents with data that is more appropriate for your deployment.
+labels:
+  template: datavirt-app-secret
```

Comment:
I am wondering if we should perhaps add the mock secrets file to a different directory  (perhaps resources/openshift/secrets instead, to have the same layout we have in application-templates

that way the admin can install all the files from resources/openshift/templates/  (they should not be installing the mock secrets )

Reply:
It may make better sense to move it into the quickstarts, since this isn't something for production and should only be used for testing.